includes/class-alg-wc-cpp-tracking.php

```diff
@@ -0,0 +1,186 @@
+<?php // phpcs:ignore
```

Comment:
@PratapDungrani As per WP standard, the class file names should be based on the class name.